docs/_sections/forms.html

```diff
@@ -87,6 +87,34 @@ <h3 id="requiredFields">Required fields</h3>
 &lt;/div&gt;
     </code></pre>
 
+    <h3 id="fieldErrors">Error state on fields</h3>
+
+    <p>Any field where user input could be deemed or returned erroneous should utilize the class <code>nypl-field-error</code>. It is up to the form designer to determine what fields should have an error state. It is up to the developer to implement the necessary front and back end validations as well as the expected input formatting that would necessitate a state change for a given field. Additionally, explanatory text must be supplied by the form designer decribing the error and how to correct it. </p>
+
+    <div class="nypl-toolkit-example">
+      <span class="nypl-example-text">Example</span>
+      <div class="nypl-text-field nypl-field-error">
+        <label for="keyword-text">Title
+          <span class="nypl-required-field">Required</span>
+        </label>
+        <input id="keyword-text" type="text" placeholder="User's erroneous input" />
+        <span class="error-desc">Please enter the correct information in the above field.</span>
+      </div>
+    </div>
+
+    <pre><code class="scss">
+.nypl-error-field
+    </code></pre>
+    <pre><code class="html">
+&lt;div class=&quot;nypl-text-field&quot;&gt;
+  &lt;label for=&quot;keyword-text&quot;&gt;Title
+    &lt;span class=&quot;nypl-required-field&nbsp;nypl-field-error&quot;&gt;Required&lt;/span&gt;
```

Comment:
this doesn't seem to match the doc... `nypl-field-error` is applied to the parent main tag (`div.nypl-text-field`) and not the one shown here